tests/integration/README.md

```
@@ -79,6 +79,35 @@ def test_example(mock_ogx_client: Any) -> None:
     mock_ogx_client.responses.create.return_value = custom_response
 ```
 
+#### `mock_conversation_store` (function-scoped)
```

markdownlint-cli2 (0.23.2)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)